sql/core/src/test/scala/org/apache/spark/sql/StringFunctionsSuite.scala

```diff
@@ -141,6 +141,19 @@ class StringFunctionsSuite extends QueryTest with SharedSQLContext {
     // scalastyle:on
   }
 
+  test("binary overlay function") {
+    // non ascii characters are not allowed in the code, so we disable the scalastyle here.
```

Comment:
This is just a confirmation; have you checked if the other databases (e.g., postgresql, oracle, ..) have the same output?

Reply:
Yeah, It is.